pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java

```diff
@@ -2195,13 +2195,13 @@ The max allowed delay for delayed delivery (in milliseconds). If the broker rece
             category = CATEGORY_STORAGE_ML,
             doc = "Minimum backlog entries for any cursor before start caching reads"
     )
-    private int managedLedgerMinimumBacklogEntriesForCaching = 1000;
+    private int managedLedgerMinimumBacklogEntriesForCaching = 100;
```

Comment:
Why do you change here ?

Reply:
This will also make it's value different with ManagedLedgerImpl

Reply:
> make ServiceConfiguration.java defaults match the defaults in broker.conf

Which is the correct value for these two fields?

Reply:
replied in other comment, https://github.com/apache/pulsar/pull/22295#issuecomment-2006237183